modules/videojsVideoProvider.js

```diff
@@ -201,13 +201,17 @@ export function VideojsProvider(providerConfig, vjs_, adState_, timeState_, call
 
   // Plugins to integrate: https://github.com/googleads/videojs-ima
   function setAdTagUrl(adTagUrl, options) {
```

Comment:
It'd be cleaner if we had a separate `setAdXml` rather than a `setAdTagUrl` that doesn't actually set the ad tag url. (same with jwplayer)